Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -319,6 +320,7 @@ public string MakeValidUsername(string username)
 
             if (user == null)
             {
+                _logger.LogInformation("Authentication request for {UserName} has been denied (IP: {IP}).", username, remoteEndPoint);
```

Comment:
Why log here but not for the other `AuthenticationException`s that could be thrown below?